### PR TITLE
Fix flaky models metadata test

### PR DESCRIPTION
**Pull request by kamilmielnik**
### Description

Example failure: https://github.com/metabase/metabase/actions/runs/11719690255

Use "hide whitespace" option for review.

### How to verify

It flakes very rarely, so I gave it 200 runs:
Stress test `#1`: https://github.com/metabase/metabase/actions/runs/11722664361
Stress test `#2`: https://github.com/metabase/metabase/actions/runs/11722668409

### CODE REVIEW

File: e2e/support/helpers/e2e-models-metadata-helpers.js [hunk 1]
```diff
@@ -34,9 +34,11 @@ export function setColumnType(oldType, newType) {
     .contains(oldType)
     .click();
 
-  cy.get(".ReactVirtualized__Grid.MB-Select").realMouseWheel({ deltaY: -200 });
-  cy.findByPlaceholderText("Search for a special type").realType(newType);
-  popover().findByLabelText(newType).click();
+  popover().within(() => {
+    cy.get(".ReactVirtualized__Grid").scrollTo(0, 0); // HACK: scroll to the top of the list. Ideally we should probably disable AccordionList virtualization
```
Comment (kamilmielnik) [inline e2e/support/helpers/e2e-models-metadata-helpers.js:38]: hack with comment copied from https://github.com/metabase/metabase/blob/f6b8e1e8a3f2ebf01f0f17530ff91e68b655fb4b/e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.js#L55

File: e2e/support/helpers/e2e-models-metadata-helpers.js [hunk 2]
```diff
@@ -34,9 +34,11 @@ export function setColumnType(oldType, newType) {
     .contains(oldType)
     .click();
 
-  cy.get(".ReactVirtualized__Grid.MB-Select").realMouseWheel({ deltaY: -200 });
-  cy.findByPlaceholderText("Search for a special type").realType(newType);
-  popover().findByLabelText(newType).click();
+  popover().within(() => {
+    cy.findByText(oldType).closest(".ReactVirtualized__Grid").scrollTo(0, 0); // HACK: scroll to the top of the list. Ideally we should probably disable AccordionList virtualization
```
Comment (kamilmielnik) [inline e2e/support/helpers/e2e-models-metadata-helpers.js:38]: Hack with comment copied from https://github.com/metabase/metabase/blob/f6b8e1e8a3f2ebf01f0f17530ff91e68b655fb4b/e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.js#L55

File: e2e/support/helpers/e2e-models-metadata-helpers.js [hunk 3]
```diff
@@ -34,9 +34,11 @@ export function setColumnType(oldType, newType) {
...
+    cy.findByPlaceholderText("Search for a special type").realType(newType);
```
Comment (uladzimirdev) [inline e2e/support/helpers/e2e-models-metadata-helpers.js:39]: why `realType` and not just `type`?

**Review by uladzimirdev — APPROVED** [review]

File: e2e/support/helpers/e2e-models-metadata-helpers.js (hunk 3 above)
Comment (kamilmielnik) [reply e2e/support/helpers/e2e-models-metadata-helpers.js:39]: I don't know, I left it as I found it (this line just got indented into `popover().within(() => {`).

**Pull request merged**